submodules/TESTER/software/buildroot/board/IObundle/iob-soc/rootfs-overlay/root/tester_verification/tester_verification.c

```diff
@@ -296,33 +294,39 @@ int main() {
 //    // Get address of string stored in SUT's memory
 //    // and invert the highest bit of MEM_ADDR_W to access the SUT's memory zone
 //    sutStr = (char *)(IOB_SOC_SUT_GET_REG5() ^ (1 << (IOB_SOC_TESTER_MEM_ADDR_W - 1)));
-//  
+//
 //    // Print the string by accessing that address
 //    uart16550_puts("[Tester]: String read from SUT's memory via shared memory:\n");
 //    for (i = 0; sutStr[i] != '\0'; i++) {
 //      uart16550_putc(sutStr[i]);
 //    }
 //    uart16550_putc('\n');
-//  
+//
 //  #ifdef USE_ILA_PFSM
 //      // Allocate memory for ILA output data
 //      const uint32_t ila_n_samples = (1<<4); //Same as buffer size
 //      uint32_t ila_data_size = ila_output_data_size(ila_n_samples, ILA0_DWORD_SIZE);
-//  
+//
 //      // Write data to allocated memory
 //      uint32_t latest_sample_index = ila_number_samples();
 //      ila_output_data(buffer, latest_sample_index, (latest_sample_index-1)%ila_n_samples, ila_n_samples, ILA0_DWORD_SIZE);
-//  
+//
 //      // Send ila data to file via UART
 //      uart16550_sendfile("ila_data.bin", ila_data_size-1, buffer); //Don't send last byte (\0)
 //  #endif
-//  
+//
   // Test iob-timer with drivers
   if (iob_timer_test() == -1){
       puts("[Tester]: iob-timer test failed!\n");
       return -1;
   }
 
+  // Test iob-gpio with drivers
+  if (iob_gpio_test() == -1){
```

Comment:
We don't have this test in the baremetal firmware, but I think it's a nice addition to the Linux verification sequence.